tests/Browser/ItemsFormTest.php

```diff
@@ -23,15 +29,62 @@ public function test_can_enter_list_of_item_ids()
         });
     }
 
-    public function test_can_submit_list_of_item_ids()
+    public function test_can_submit_list_of_item_ids_not_present_in_db()
+    {
+
+        $this->browse(function (Browser $browser) {
+
+            $import = ImportMeta::factory()
+            ->for(User::factory()->uploader())
+            ->create();
+
+            Mismatch::factory(2)
+                ->for($import)
+                ->create();
+            
+            $mismatch = Mismatch::first();
+            $mismatch2 = Mismatch::find(2);
+
+            $item1_in_db_id =  $mismatch->item_id;
+            $item2_in_db_id =  $mismatch2->item_id;
+            $item1_not_in_db_id = 'Q' . substr($item1_in_db_id, 1) . '000';
+            $item2_not_in_db_id = 'Q' . substr($item2_in_db_id, 1) . '000';
+
+            $browser->visit(new HomePage)
+                    ->keys('@items-input', $item1_not_in_db_id, '{return_key}', $item2_not_in_db_id)
+                    ->press('button')
+                    ->waitFor('@not-found')
+                    ->assertTitle('Mismatch Finder - Results')
+                    ->assertSee('[ "'. $item1_not_in_db_id . '", "'. $item2_not_in_db_id .'" ]');
+        });
+    }
+
+    public function test_can_submit_list_of_item_ids_present_in_db()
     {
+
         $this->browse(function (Browser $browser) {
+
+            $import = ImportMeta::factory()
+            ->for(User::factory()->uploader())
+            ->create();
+
+            Mismatch::factory(2)
+                ->for($import)
+                ->create();
+            
+            $mismatch = Mismatch::first();
+            $mismatch2 = Mismatch::find(2);
+
+            $item1_in_db_id =  $mismatch->item_id;
+            $item2_in_db_id =  $mismatch2->item_id;
```

Comment:
feel free to change it ;)

Reply:
I would also be with @Silvan-WMDE on this. But this means that we need some kind of linter rule for it, and since we don't have one, I think it's fine for now.

Reply:
i subconsciously emulated the naming convention of the test functions. I don't really know what the convention for multi-variable names in laravel is.